demostat/templates/demostat/components/demo_card.html

```diff
@@ -1,6 +1,6 @@
 <div class="card {% if demo.is_next %} border-success {% endif %}">
   <div class="card-body">
-    <h5 class="card-title">{{ demo.title }} <small class="text-muted">von <a href="{% url 'demostat:organisation' demo.organisation.slug %}">{{ demo.organisation.name }}</a></small></h5>
+    <h5 class="card-title">{{ demo.title }} {% if demo.organisation.all %}<small class="text-muted">von {% if demo.organisation.all|length > 1 %} {{ demo.organisation.all|length }} Organisationen{% else %}<a href="{% url 'demostat:organisation' demo.organisation.all.0.slug %}">{{ demo.organisation.all.0.name }}</a>{% endif %}</small>{% endif %}</h5>
     <h6 class="card-subtitle mb-2 text-muted">{{ demo.date|date:"d.m.Y H:i" }} Uhr, {{ demo.location.name }}</h6>
     <p class="card-text">{{ demo.description }}</p>
```

Comment:
```suggestion
    <p class="card-text">{{ demo.description|truncatewords:20 }}</p>
```
Hier würde könnte ein [truncatewords](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#truncatewords) anstatt der kompletten Beschreibung anzeigen. Dies ist ist durch die begrenzte Breite angemessen. Hier wäre eine Wortzahl von 30-40 Wort angemessen

Reply:
Meiner Meinung nach sollte die Beschreibung auf auf der Karte so lang wie nötig sein. Alles andere kommt in die Anmerkungen.
Es geht nicht darum, dass man noch extra auf den Link klicken muss, sondern darum, dass man schnell informiert ist. Nachrichtenportale machen das gerne, weil sie mit jedem Seitenaufruf neue Werbung veröffentlichen können. Wir brauchen das nicht.